src/Azure/Orleans.Streaming.EventHubs/Providers/Streams/EventHub/EventHubCheckpointer.cs

```diff
@@ -80,7 +85,7 @@ private EventHubCheckpointer(AzureTableStreamCheckpointerOptions options, string
             }
             persistInterval = options.PersistInterval;
             dataManager = new AzureTableDataManager<EventHubPartitionCheckpointEntity>(options.TableName, options.ConnectionString, loggerFactory);
```

Comment:
add a log line here to say: created a checkpointer with streamProviderName xxx, partition xxxx, serviceId xxx, so if users configured their serviceId wrong, or isn't aware of that serviceId is part of partition now, we can ask them to send us their logs and see what exactly is used as partition